src/runtime/StaticStrings.h

```diff
@@ -799,12 +799,23 @@ namespace Escargot {
 #define FOR_EACH_LAZY_THREADING_STATIC_STRING(F)
 #endif
 
+#define ESCARGOT_ASCII_TABLE_MAX 256
+#define ESCARGOT_STRINGS_NUMBERS_MAX 128
+
 class StaticStrings {
 public:
     StaticStrings(AtomicStringMap* atomicStringMap)
         : dtoaCacheSize(5)
         , m_atomicStringMap(atomicStringMap)
     {
+        asciiTable = new (malloc(sizeof(AtomicString) * ESCARGOT_ASCII_TABLE_MAX)) AtomicString[ESCARGOT_ASCII_TABLE_MAX];
+        numbers = new (malloc(sizeof(AtomicString) * ESCARGOT_STRINGS_NUMBERS_MAX)) AtomicString[ESCARGOT_STRINGS_NUMBERS_MAX];
```

Comment:
Regarding memory allocation, why `malloc` is used here instead of just new operator?
To avoid any constructor/destructor call?

Reply:
AtomicString inherits gc class so it uses gc_malloc
but we don't need to store AtomicStriong on gc-allocated memory
that's why I use libc malloc, free